README.md

```diff
@@ -16,6 +16,16 @@ The `supa_audit` PostgreSQL extension is a generic solution for tracking changes
 
 The audit table, `audit.record_version`, leverages each records primary key values to produce a stable `record_id::uuid`, enabling efficient (linear time) history queries.
 
+## Cloud Extensibility
+Since the result of this project is a SQL-type extension, it can be easily adapted for PostgreSQL in the cloud (DBaaS). Just download the most recent SQL file (currently, **supa_audit--0.2.3.sql**). Copy this file to a new file and edit it putting this statement line at the top:
```

Comment:
Suggest removing the name of the file -- it's bound to get out of date and will become confusing at that point

Reply:
Sure, just remove every thing in the parenthesis.